extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -204,8 +207,7 @@ require([
 				if (hasUpvoted) {
 					$svg.attr('class', 'embeddable-discussions-upvote-icon');
 					event.currentTarget.setAttribute('data-hasUpvoted', '0');
```

Comment:
I'd use only jquery function ($.attr) - it's better to keep one way for setting attributes.